### ISSUE TITLE
Unable to connect "protocol security negotiation failure"

### ISSUE THREAD

**Issue by remminaheat** [FreeRDP/Remmina#453]
Hello, since yesterday Remmina was working fine, on my ubuntu 14.04 with RDP to a windows 2008 server, with a vpn using tcp.

since this morning the connection is not working anymore, 
i have made no changes (only run updates from software updater). I ve already read tons of posts that talk about freerdp issues, then suggest to remove Know hosts file or change security method (rdp,nla,negotiate). Nothin worked. I ve tried to reinstall remmina and restart network services, then for just one time Remmin can connect, but just for one time, because from the second time it come back to error "protocol security negotiation failure".

Basically i m going crazy because i really need to use remmina, and i m reinstalling ubuntu once a day, so i hope somebody understood what is the problem and can give me some help. thanks a lot in advance.

the last error message i m getting:
[15:05:23:015] [6832:-89577728] [ERROR][com.freerdp.core.connection] - Error: protocol security negotiation or connection failure

and this before start the connection:
(remmina:6885): libappindicator-WARNING **: Unable to get the session bus: Could not connect: Connection refused

**Comment by giox069** [comment]
Are you using the latest version of remmina in the "next" PPA for ubuntu ?

**Comment by remminaheat** [comment]
from help i saw i m using: 1.2.0-rcgit.2

**Comment by giox069** [comment]
According the version you reported, and the fact that you updated something, yes, you are using the latest version available on the PPA. It's a bit difficult to reproduce now for me. In the meanwhile, have you tryed to connect via xfreerdp commandline to the same server ?
``` xfreerdp /v:servername /u:username /d:domain```
(I hope this is the correct syntax)

**Comment by remminaheat** [comment]
just tried and getting the same error:  [ERROR][com.freerdp.core.connection] - Error: protocol security negotiation or connection failure, (btw xfreedrp was not installed (just intalled before trying)

**Comment by remminaheat** [comment]
if i start remming from terminal with sudo i get this message, then the remmina window show up:
(remmina:7457): Gtk-WARNING **: Unknown property: GtkStatusbar.margin-start
(remmina:7457): Gtk-WARNING **: Unknown property: GtkStatusbar.margin-end

**Comment by giox069** [comment]
Do not use sudo, it is not necessary and not tested. Start remmina as a normal user.
The problem seems to be in freerdp libraries. Can you execute
```dpkg --list "freerd*"```
and paste the output here ?

**Comment by giox069** [comment]
Please use this, I wrote the wrong command:
```dpkg --list "*freerd*"```

**Comment by remminaheat** [comment]
~$ dpkg --list "*freerd*"
Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name           Version      Architecture Description
+++-==============-============-============-=================================
un  freerdp        <none>       <none>       (no description available)
ii  freerdp-x11    1.2.0~git201 amd64        RDP client for Windows Terminal S
ii  libfreerdp-plu 1.2.0~git201 amd64        RDP client for Windows Terminal S
ii  libfreerdp1:am 1.2.0~git201 amd64        RDP client for Windows Terminal S
un  lightdm-remote <none>       <none>       (no description available)

**Comment by giox069** [comment]
... but really: if also freerdp fails, this is not a Remmina problem.
Check your network connectivity, ping the server, try to telnet the server on port 3389 (telnet servername 3389 [ENTER], and your linus should show you a "Connected to server... Escape character is...").

**Comment by remminaheat** [comment]
its holding on trying... so is not connected and that explain why sometime after network manager restart remmina come back to work... 
but connection is working right now as i writing you from firefox, so i really don't kwno what to do... i m not really good with networking.

**Comment by giox069** [comment]
I'm closing this issue because it's not remmina related. I will be able to help you later this evening/night (central european time) if you leave me an email at giodev (at) panozzo.it.
